# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libotr/hotfix-socket.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libotr-4.1.1/tests/regression/client/client.c.vanilla	2022-08-25 00:36:08.008861228 +0200
+++ libotr-4.1.1/tests/regression/client/client.c	2022-08-25 00:36:51.788864742 +0200
@@ -27,6 +27,7 @@
 #include <syscall.h>
 #include <sys/epoll.h>
 #include <sys/types.h>
+#include <sys/socket.h>
 #include <sys/un.h>
 #include <unistd.h>
 
